client/src/pages/onboarding/cloud-provider.tsx

```diff
@@ -27,11 +27,12 @@ const CloudProvider = () => {
   const [
     saveDigitalOceanAccessTokenMutation,
   ] = useSaveDigitalOceanAccessTokenMutation();
-  const digitalOceanAccessTokenSchema = Yup.object().shape({
-    token: Yup.string()
+  const digitalOceanAccessTokenSchema = yup.object().shape({
+    token: yup
+      .string()
       .trim()
-      .min(64, 'Token too short, it should be exactly 64 chars long')
-      .max(64, 'Token too long, it should be exactly 64 chars long')
+      .min(64, 'Token too short, it should be exactly 64 characters long')
```

Comment:
@Akirtovskis the regex already say that the string should be 64 chars long, so is this really needed?

Reply:
@pradel idea behind this was to give better feedback, when for example you paste incomplete token or enter something afterwards, but could be also removed. What do you think?

Reply:
Yeah make sense 👍